client/src/shared/sdk/telemetry.ts

```diff
@@ -78,6 +78,7 @@ export interface DataResponse extends CanData {
     rx0: PowerBoard_RX0;
     rx1: PowerBoard_RX1;
   };
+  calculated: {};
```

Comment:
Why is this needed? Do you plan to return real time calcaulted data later?

Reply:
fixed in latest commit, just added a separate interface rather than tacking on.

Reply:
I did this in a different way (that works on client), but now am reminded why I did it this way. The backend build is failing now, if you check the workflow run. Let me know if there's a better way to avoid this error, other than putting calculated in DataResponse.
Error: `Object literal may only specify known properties, and 'calculated' does not exist in type 'DataResponse'.`

Reply:
When you remove it from the schema here, you also need to change the API response which is using this schema to remove the `calculated` key here: https://github.com/Solar-Gators/Pit-GUI/blob/4eaff788de5f97bb9ca8937b2d2e7bd2bb86c87b/backend/controllers/live.server.controller.ts#L68

I made the response of type `DataResponse` to ensure that the response shape always matches the `DataResponse` schema. Static analysis is great :)

https://github.com/Solar-Gators/Pit-GUI/blob/4eaff788de5f97bb9ca8937b2d2e7bd2bb86c87b/backend/controllers/live.server.controller.ts#L27